packages/wonder-blocks-tooltip/index.js

```diff
@@ -1,5 +1,9 @@
 // @flow
+import type {Placement} from "./util/types.js";
+
 import Tooltip from "./components/tooltip.js";
 import TooltipContent from "./components/tooltip-content.js";
 
 export {Tooltip as default, TooltipContent};
+
+export type {Placement};
```

Comment:
Feels weird to make this part of the exported Tooltip package API. It's a commitment to support it as part of the API even if popover eventually doesn't need it. That said, this type is part of the API anyway, given that it's used by the exported components.

However, I think if we start exporting anything else from there for use in popover, we should really reconsider how we share code as we're committing to maintaining that API surface long term.